Dockerfile-alpine.template

```diff
@@ -92,7 +92,8 @@ RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg tar \
   && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
   && apk del .build-deps-yarn \
   # smoke test
-  && yarn --version
+  && yarn --version \
+  && rm -rf /tmp/*
```

Comment:
It probably makes sense to move this above the smoke test to where the other cleanups are happening

Reply:
That's a good idea that I missed!

Reply:
moving the cleaning before the yarn --version keep tmp files created by yarn

```
$ docker run -it --rm docker-node:18-alpine3.19 sh

/ # ls -alh /tmp/
total 8K     
drwxr-xr-x    2 root     root        4.0K Apr  8 09:21 .
drwxr-xr-x    1 root     root        4.0K Apr  8 09:21 ..
/ # yarn --version
1.22.19
/ # ls -alh /tmp/
total 12K    
drwxr-xr-x    3 root     root        4.0K Apr  8 09:21 .
drwxr-xr-x    1 root     root        4.0K Apr  8 09:21 ..
drwxr-xr-x    3 root     root        4.0K Apr  8 09:21 v8-compile-cache-0

```

Reply:
Oh, didn't notice that, sorry! In that case, we should keep it!